src/auth.js

```diff
@@ -2,17 +2,17 @@ import { getTokenExpiresAtDate } from './utils.js';
 import { parseResponse } from './response.js';
 
 let fetch;
-try {
-  fetch = require('node-fetch'); // eslint-disable-line global-require
-} catch (Exception) {
+if (typeof window !== 'undefined') {
```

Comment:
Should we also check for the existence of fetch? I'm just wondering if there's a case where someone could have have the `window` object via a bundler, but no `fetch` and we'd just error out in that case.